doc/contribute/guidelines.rst

```diff
@@ -536,12 +536,28 @@ reference manuals, etc.
 Coding Style
 ============
 
+.. note::
+   Coding style is enforced on any new or modified code, but contributors are
+   not expected to correct the style on existing code that they are not
+   modifying.
+
+.. note::
+   For style aspects where the guidelines don't offer explicit guidance or
+   permit multiple valid ways to express something, new code should try to follow
```

Comment:
"should" or "shall"?

Reply:
> "should" or "shall"?

Does anyone have an "American English" dictionary? :-)

Reply:
It's not a dialect thing, but in this context since it's followed by "try to", then "should" I think fits, since shall would express determination and contradict the rest of the phrase. Maybe the suggestion here is to replace "should try to" entirely with "shall", I'm not sure I'm what case a style can't be followed and saying to just try is ambiguous as far as how to enforce this guideline

Reply:
I am fine with "must".